airgun/views/computeresource.py

```diff
@@ -174,6 +178,8 @@ class virtual_machines(SatTab, SearchableViewMixin):
                 'Power': Text('.//span[contains(@class,"label")]'),
             }
         )
+        more_button = Text('//span[@class="caret"]')
```

Comment:
You can try different solutions here:
1) Use `ActionsDropdown` widget directly
2) Inherit `ActionsDropdown` class for your own custom class implemented here locally and override `dropdown` or `button` elements or whatever you need
3) If neither solution works for you - implement your own widget in that module that will work with dropdown element as with dropdown, not separate buttons and etc